modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -460,6 +462,69 @@ public void testPostCustomObjectEntryWithNestedCustomObjectEntriesInOneToManyRel
 			_OBJECT_FIELD_NAME_2, _NEW_OBJECT_FIELD_VALUE_2);
 	}
 
+	@Test
+	public void testPostCustomObjectEntryWithWrongNestedCustomObjectEntriesInRelationship()
+		throws Exception {
+
+		try (LogCapture logCapture = LoggerTestUtil.configureLog4JLogger(
+				"com.liferay.portal.vulcan.internal.jaxrs.exception.mapper." +
+					"WebApplicationExceptionMapper",
+				LoggerTestUtil.WARN)) {
+
+			_objectRelationship =
```

Comment:
You are overriding the value of this attribute with different relationships. The problem is in the tearDown method, you are only deleting the last instance, the rest are lost.

Reply:
I'm thinking of dividing `testPostCustomObjectEntryWithWrongNestedCustomObjectEntriesInRelationship` into two tests:

-  `testPostCustomObjectEntryWithWrongNestedCustomObjectEntriesInManyToManyRelationship`. A Many to Many relationship is created and the tearDown will be executed and delete the relationship attribute. There will be a test case
-  `testPostCustomObjectEntryWithWrongNestedCustomObjectEntriesInOneToManyRelationship`. A One to Many relationship is created and we can test the two wrong use cases with only one relationship

Reply:
Ok